app/assets/stylesheets/components/_banner.scss

```diff
@@ -2,3 +2,45 @@
   background-color: #fff;
   box-shadow: inset 0 -1px 0 0 #ccc;
 }
+
+.usa-banner__inner {
+  padding-left: 0;
+  padding-right: 0;
+}
+
+.usa-banner__header {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+  + .usa-banner__content.usa-accordion__content {
+    position: relative;
+  }
+}
+
+.close-banner-how-you-know {
+  display: none;
+}
+
+@media #{$mobile} {
+  .usa-banner__header--expanded .usa-banner__inner {
+    margin-left: auto;
+  }
+
+  .usa-banner__button[aria-expanded=true]::after {
+    display: none;
+  }
+
+  .close-banner-how-you-know {
+    background: url(#{$image-path}/close-primary.svg) top left no-repeat;
```

Comment:
I noticed a console error in unrelated testing in the development environment, I suspect is caused by this.

>[Error] Failed to load resource: the server responded with a status of 403 () (close-primary.svg, line 0)

Based on how images are referenced in other stylesheets, I suspect this should be `image-path` helper?

```suggestion
    background: url(image-path('close-primary.svg')) top left no-repeat;
```

Reply:
I tried this, but it doesn't fix it. Does the image exist?

Reply:
It's in the identity-style-guide:

```
identity-site:master> find node_modules -name '*close-primary.svg' 
node_modules/identity-style-guide/dist/assets/img/close-primary.svg
```

Reply:
shucks...it worked locally, but clearly something is amiss...easy fix would be to copy `node_modules/identity-style-guide/dist/assets/img/close-primary.svg` into the path referenced by the `image-path` helper, but I'd still be left how the `dist` assets are supposed to be reached...

In any event, good catch @mitchellhenke

Reply:
I get `GET http://localhost:3000/images/close-primary.svg 404 (Not Found)` with it locally

Reply:
It gets copied to:

```
identity-site:master> find _site -name '*close-primary.svg'
_site/assets/img/close-primary.svg
```

Reply:
This may get it:
```
background: url(../img/close-primary.svg) top left no-repeat;
```
we'll see...

Reply:
After `git pull`ing on master, I can't get the project to run anymore.  Seems like the `webpack` build is not quite working...stay tuned...

Reply:
Continually getting this:
<img width="753" alt="Screen Shot 2020-09-14 at 12 07 34 PM" src="https://user-images.githubusercontent.com/1053255/93110132-ea947280-f682-11ea-9cc0-988e59b7edd2.png">

Reply:
Since the current code bombs, I'm still going to push the change and see what happens in CI...

Reply:
https://github.com/18F/identity-idp/pull/4171 may be the solution.  Draft PR built successfully.